br/pkg/lightning/backend/local/local.go

```diff
@@ -1531,19 +1531,18 @@ loopWrite:
 					continue
 				}
 			}
+			if err != nil {
```

Comment:
if error is because of suspended, should we try to add more backoff time as we know that it can take some time. It will prevent unnecessary re writes.

Reply:
this pr only tries to fix the bug

for the Pause Ingest introduced by EBS BR, we need to handle it in whole later